test/monitors/expected/br_return.wasm.branches-c-.out

```diff
@@ -1,12 +0,0 @@
-func "main":
```

Comment:
I manually removed all expected output from the folder before running the `update-expected` script. It's because a bunch of the test wasm modules were removed from this folder at some point, so the expected output files aren't used anymore, and the update script only writes out a file for each (test module, monitor setting) pair, instead of removing the entire `expected` directory and start fresh.